runtime/testy/studentCode.py

```diff
@@ -102,13 +113,11 @@ def createKey_main():
   restarts = Robot.getValue("Restarts")
   Robot.setValue(restarts+1, "Restarts")
 
-def hibikeSubscribeDevice_setup():
-  pass
+# def hibikeSubscribeDevice_setup():
```

Comment:
I would prefer you rename these to end in `_setup_DISABLED`/`_main_DISABLED` rather than commenting out (so at least they get syntax checked). Or hide them behind an `if/else`. Or delete them entirely.